scania-digital-design-system/tegel
# test(footer): Adding unit tests

**Describe pull-request**  
Adding unit tests for footer component

**Solving issue**  
Fixes: [CDEP-3004](https://tegel.atlassian.net/browse/CDEP-3004)

**How to test**  
_Add description how to test if possible_
1. Checkout branch
2. Check so that `npm run tests` goes thru


[CDEP-3004]: https://tegel.atlassian.net/browse/CDEP-3004?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
